src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+          action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+    >
+
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+        <label class="form-field">
+        Surname:
+          <input type="text"
+                 name="surname"
+                 required
+          >
+        </label>
```

Comment:
```suggestion
        <label class="form-field">
          Surname:
          <input
            type="text"
            name="surname"
            required
          >
        </label>
```
разметка должна быть отформатированна вот так, пофикси для всех тегов на странице